examples/account_sample/src/ops/mod.rs

```diff
@@ -15,8 +15,8 @@ pub struct Client {
 }
 
 impl Client {
-    pub fn new(params: RequestParams) -> Result<Self> {
-        let client = SyncClientBuilder::from_params(params).build()?;
+    pub fn new(address: &'static str) -> Result<Self> {
+        let client = SyncClientBuilder::new().static_nodes(vec![address]).build()?;
```

Comment:
I think it would be convenient to have a `static_node` method too that accepts just a single address.